app/src/main/java/org/wikipedia/settings/dev/playground/CategoryDeveloperPlayGroundViewModel.kt

```diff
@@ -135,7 +137,7 @@ class CategoryDeveloperPlayGroundViewModel : ViewModel() {
             L.e(throwable)
         }) {
             try {
-                AppDatabase.instance.categoryDao().deleteAll()
+                deleteOldDataInBatches(System.currentTimeMillis(), 100)
```

Comment:
Any reason why you changed to use `deleteOldDataInBatches` instead of `deleteAll`?